import-export-cli/integration/environmnet_test.go

```diff
@@ -36,47 +40,97 @@ func TestListEnvironments(t *testing.T) {
 	assert.Contains(t, response, apim.GetEnvName(), "TestListEnvironments Failed")
 }
 
+func validateExportDirectoryIsChanged(t *testing.T, args *setTestArgs) {
+	t.Helper()
+	output, _ := environmentSetExportDirectory(t, args)
+	base.Log(output)
+	assert.Contains(t, output, "Export Directory is set to", "Export Directory change is not successful")
+}
+
+func validateExportApisPassed(t *testing.T, args *initTestArgs) {
+	t.Helper()
+	output, error := exportApisWithOneCommand(t, args)
+	assert.Nil(t, error, "Error while Exporting APIs")
+	assert.Contains(t, output, "export-apis execution completed", "Error while Exporting APIs")
+
+	//Derive exported path from output
+	exportedPath := base.GetExportedPathFromOutput(strings.ReplaceAll(output, "Command: export-apis execution completed !", ""))
+	count, _ := base.CountFiles(exportedPath)
+	assert.Equal(t, 1, count, "Error while Exporting APIs")
+}
+
 //Change Export directory using apictl and assert the change
 func TestChangeExportDirectory(t *testing.T) {
-	apim := apimClients[0]
-	changedExportDirectory := utils.MockTestExportDirectory + utils.DefaultExportDirName
+	dev := apimClients[0]
+	changedExportDirectory, _ := filepath.Abs(utils.CustomTestExportDirectory + utils.DefaultExportDirName)
 	defaultExportPath := utils.DefaultExportDirPath
 
 	args := &setTestArgs{
-		srcAPIM:             apim,
+		srcAPIM:             dev,
 		exportDirectoryFlag: changedExportDirectory,
 	}
-	output, _ := environmentSetExportDirectory(t, args)
-	base.Log(output)
+	validateExportDirectoryIsChanged(t, args)
+
+	apim := apimClients[0]
+	projectName := "OpenAPI3Project"
+	username := superAdminUser
+	password := superAdminPassword
+
+	apiArgs := &initTestArgs{
+		ctlUser:   credentials{username: username, password: password},
+		srcAPIM:   apim,
+		initFlag:  projectName,
+		oasFlag:   utils.TestOpenAPI3DefinitionPath,
+		forceFlag: false,
+	}
+
+	//Assert that project import to publisher portal is successful
+	validateImportInitializedProject(t, apiArgs)
+
+	base.RemoveDir(projectName)
```

Comment:
Can we move this _base.RemoveDir()_ into the _validateImportInitializedProject(_) function and register it as a t.Cleanup()? It is an essential cleanup step when running this scenario but it is a low level detail that is not required to be known at the high level of the test case.